.github/workflows/kserve-test.yml

```diff
@@ -0,0 +1,128 @@
+name: "Kserve Smoke Test"
+
+on:
+  schedule:
+    - cron: "20 4 * * 1" # once a week
+  workflow_dispatch:
+
+  # push:
+  #   branches: [main]
+  #   paths:
+  #     - "!docs"
+  #     - "!demo"
+
+  # pull_request:
+
+env:
+  KIND_CLUSTER_NAME: "kserve-testing"
+  ISVC_NAME: "caikit-tgis-isvc"
+
+jobs:
+  smoke-test:
+    name: Kserve Smoke Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Free Disk Space
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          tool-cache: false
+
+          android: true
+          dotnet: true
+          haskell: true
+          large-packages: true
+          swap-storage: true
+
+      - name: Create k8s Kind Cluster
+        uses: helm/kind-action@v1.5.0
+        with:
+          cluster_name: ${{ env.KIND_CLUSTER_NAME }}
+          config: test/kserve/kind_config.yaml
+
+      - name: Install kserve
+        run: |
+          # Use the hack/quick_install.sh script from the kserve repo
+          curl https://raw.githubusercontent.com/opendatahub-io/kserve/v0.11.1.1/hack/quick_install.sh | bash
```

Comment:
@danielezonca  I just realized that `quick_install.sh` references `kserve/kserve`, making it kinda pointless to use the odh fork. Wdyt?